assets/sass/local.scss

```diff
@@ -1,7 +1,3 @@
-.govuk-main-wrapper a:not(.govuk-button, .govuk-tabs__tab, .bapv-timetable-dates__date--selected>a, .moj-pagination__link, .govuk-error-summary a):visited,
```

Comment:
Setting class directly on `<a>` tags instead to remove visited state colour; less error-prone.